src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/steps/CoverageBuildAction.java

```diff
@@ -69,32 +59,18 @@ public final class CoverageBuildAction extends BuildAction<Node> implements Stap
     private final QualityGateResult qualityGateResult;
 
     private final String icon;
-    // FIXME: Rethink if we need a separate result object that stores all data?
     private final FilteredLog log;
 
     /** The aggregated values of the result for the root of the tree. */
     private final List<? extends Value> projectValues;
 
-    /** The delta of this build's coverages with respect to the reference build. */
-    private NavigableMap<Metric, Fraction> difference;
-
     /** The coverages filtered by modified lines of the associated change request. */
     private final List<? extends Value> modifiedLinesCoverage;
 
-    /** The coverage delta of the associated change request with respect to the reference build. */
-    private NavigableMap<Metric, Fraction> modifiedLinesCoverageDifference;
-
-    /** The coverage of the modified lines. */
-    private final List<? extends Value> modifiedFilesCoverage;
-
-    /** The coverage delta of the modified lines. */
-    private NavigableMap<Metric, Fraction> modifiedFilesCoverageDifference;
-
     static {
         CoverageXmlStream.registerConverters(XSTREAM2);
         registerMapConverter("difference");
```

Comment:
These two `registerMapConverter`s are also not needed, since we've already removed the fields.